index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Use imperative tense and capitalizing for your commit messages going forward e.g. Add or Edit instead of Added or Edits

Reply:
Thank you. I have made all changed required